server/webhook_http_test.go

```diff
@@ -344,6 +344,13 @@ func TestWebhookHTTP(t *testing.T) {
 			ExpectedText:     "",
 			CurrentInstance:  true,
 		},
+		"SERVER issue comment created indentation": {
```

Comment:
Do you mind putting this test next to the other "SERVER issue comment created" test?